### PR TITLE
Remove isClusterMode configuration

### PR DESCRIPTION
**Pull request by HTHou**
## Description

isClusterMode is useless in the current version of IoTDB, this PR is for removing it.

### CODE REVIEW

File: iotdb-core/tsfile/src/main/java/org/apache/iotdb/tsfile/utils/TsFileUtils.java [hunk 1]
```diff
@@ -55,4 +55,9 @@ public static boolean isSequence(File tsFile) {
     File folder = tsFile.getParentFile().getParentFile().getParentFile().getParentFile();
     return folder.getName().equals("sequence");
   }
+
+  public static void main(String[] args) {
+    String s = "https://www.apache.org/licenses/";
+    System.out.println(s.matches("http?://.*apache.org/licenses/?"));
+  }
```
Comment (HTHou) [inline iotdb-core/tsfile/src/main/java/org/apache/iotdb/tsfile/utils/TsFileUtils.java:62]:
```suggestion

```

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java [hunk 2]
```diff
@@ -237,8 +237,7 @@ protected void doAddNode() {
   /** Prepare cluster IoTDB-DataNode */
   private boolean prepareDataNode() throws StartupException, IOException {
     long startTime = System.currentTimeMillis();
-    // Set cluster mode
-    config.setClusterMode(true);
+    //    config.checkMultiDirStrategyClassName();
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java:240]: .

Comment (HTHou) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java:240]:
```suggestion
```

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**